interfaces/dbus/backend.go

```diff
@@ -132,6 +156,68 @@ func addContent(securityTag string, executableSnippets [][]byte, content map[str
 	}
 }
 
+func (b *Backend) setupBusservices(snapInfo *snap.Info, snippets map[string][][]byte) error {
+	snapName := snapInfo.Name()
+
+	content := map[string]*osutil.FileState{}
+	for _, appInfo := range snapInfo.Apps {
+		securityTag := appInfo.SecurityTag()
+
+		// FIXME: layer violation, we really should do that in the
+		//        dbus interface but without PR #2613 its really tricky
+		for _, slot := range appInfo.Slots {
+			if slot.Interface != "dbus" {
+				continue
+			}
+			bus, ok := slot.Attrs["bus"].(string)
+			if !ok {
+				continue
+			}
+			if bus != "session" {
```

Comment:
I think we want that in the longer run, my main focus for this PR was to unblock people who need dbus activated services (I think unity and kde has some of those).